make/autoconf/flags-cflags.m4

```diff
@@ -575,10 +575,19 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_HELPER],
   elif test "x$TOOLCHAIN_TYPE" = xmicrosoft; then
     # The -utf-8 option sets source and execution character sets to UTF-8 to enable correct
     # compilation of all source files regardless of the active code page on Windows.
```

Comment:
Seems like this comment should be updated and moved near the new code block for setting up `CHARSET_CFLAGS`.